aspnetcore/signalr/java-client.md

```diff
@@ -69,6 +69,19 @@ SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
 
 This can safely be ignored.
 
+
+## Configure access token provider
+
+In the SignalR Java client, you can configure a bearer token to use for authentication by providing an "access token factory" to the HttpHubConnectionBuilder. Use `withAccessTokenFactory` to provide an RxJava [Single<String>](http://reactivex.io/documentation/single.html). With a call to [Single.defer](http://reactivex.io/RxJava/javadoc/io/reactivex/Single.html#defer-java.util.concurrent.Callable-), you can write logic to produce access tokens for your client.
```

Comment:
Code-fence HttpHubConnectionBuilder.

Reply:
Actually, we should link to the API doc here if possible. Do we code-fence links to API docs? I've normally see that we don't.